WelsonJS.Toolkit/WelsonJS.Launcher/AssemblyLoader.cs

```diff
@@ -355,7 +371,7 @@ private static void DownloadFile(string url, string dest)
             }
             catch (Exception ex)
             {
-                Logger.Error("Error downloading {0}: {1}", url, ex.Message);
+                Logger?.Error("Error downloading {0}: {1}", url, ex.Message);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Potential log injection detected. Input is being passed directly to log statements, which could allow attackers to inject malicious content into log files. This can lead to log forging, data integrity issues, or even remote code execution in some cases. Sanitize all external input before logging, especially data from HTTP requests. Use encoding functions like HtmlEncode() or validated libraries to properly sanitize the input. Learn more - https://cwe.mitre.org/data/definitions/117.html